### PR TITLE
[knx] Update README.md

### PR DESCRIPTION
**Pull request by Dirk68**
[KNX] 
Binding documentation improvement
I added two notes which would have saved me hours when switching from KNX1 to KNX2.
See https://community.openhab.org/t/knx2-massive-problems/67309/6
Regards,
Dirk

### CODE REVIEW

**Comment by Dirk68** [comment]
> 
> 
> Please remove those `<br>` tags.
> We don't use html tags in readme files unless they are necessary like for nested tables.
> 
> The website rendering will take care of the view when the build process is triggered.

Sorry, I've got no experience with the conformal synthax. But how do I achieve not to have the notes in the running text but in separate lines when not using <br tags?

**Comment by Confectrician** [comment]
For the table i have indeed no idea for now, but i am not sure if newlines in a markdown table look good in general. Something we should test.
Leave them there and we will have a look what the website build will bring us.
We can revert it if it doesn't look well.

For the second part you could transform the note to an independent paragraph by adding a blank line.
I will add a suggestion below.

File: addons/binding/org.openhab.binding.knx/README.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ Different kinds of channels are defined and can be used to group together Group
 
 #### Standard Channel Types
 
-Standard channels are used most of the time. They are used in the common case where the physical state is owned by a decive within the KNX bus, e.g. by a switch actuator who "knows" whether the light is turned on or of or by a temperature sensor which reports the room temperature regularly.
+Standard channels are used most of the time. They are used in the common case where the physical state is owned by a decive within the KNX bus, e.g. by a switch actuator who "knows" whether the light is turned on or of or by a temperature sensor which reports the room temperature regularly.<br/><br/>
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.knx/README.md]:
```suggestion
Standard channels are used most of the time. They are used in the common case where the physical state is owned by a decive within the KNX bus, e.g. by a switch actuator who "knows" whether the light is turned on or of or by a temperature sensor which reports the room temperature regularly.

```

Comment (Dirk68) [reply addons/binding/org.openhab.binding.knx/README.md]: Agreed, thanks for your help.

File: addons/binding/org.openhab.binding.knx/README.md [hunk 2]
```diff
@@ -66,6 +66,8 @@ Different kinds of channels are defined and can be used to group together Group
 
 Standard channels are used most of the time. They are used in the common case where the physical state is owned by a decive within the KNX bus, e.g. by a switch actuator who "knows" whether the light is turned on or of or by a temperature sensor which reports the room temperature regularly.
 
+Note: After changing the DPT of already existing Channels, openHab needs to be restarted for the changes to become effective.
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.knx/README.md]: I'm a little bit late, but please spell openHAB with the correct capitals.

Comment (Dirk68) [reply addons/binding/org.openhab.binding.knx/README.md]: Hi Martin,
thanks for the hint - should be OK now.
Dirk

**Comment by martinvw** [comment]
@Confectrician all fine for you?

**Pull request merged**